### DISCUSSION TITLE
How to use multiconfig.lua ?

### DISCUSSION THREAD

**Discussion by ParkerEde** [pascallanger/DIY-Multiprotocol-TX-Module#962] [Q&A] [answered]
I would like to give my two TX16S 4in1 the same ID so that the connection to the Rx is the same.
There is the multiconfig.lua for this. Unfortunately I can't find a "how to" anywhere. I don't really have the confidence to do this. What do I have to change? 
There is the "Global ID" and the "Cyrf ID". Do I have to change both? And once you have changed the IDs, do you have to click on "Format EEPROM" to confirm the whole thing?

**Comment by pascallanger** [comment, marked as answer]
Just make both "Global ID" and the "Cyrf ID" the same, nothing else...

**Reply by ParkerEde** [reply]
okay, thanks. Just to be sure, format EEPROM does not have to be executed separately?

**Reply by pascallanger** [reply]
Format eeprom = factory reset
It will erase everything
You've never formatted a drive or USB stick or sd card ?

**Reply by ParkerEde** [reply]
okay. thank you.

**Reply by ParkerEde** [reply]
I have certainly already formatted drives. But what format means with an HF module is not clear to me. It could have been that formatting had to be carried out with the new IDs so that they could then be adopted. Maybe I was thinking too complicated.
